adds stewardship blog post (#82)
datatogether/website: content/posts/04_stewardship.md

---
title: "Discussion: Stewardship (August 2019)"
tagline: "A Data Together 2019 Reading Group Discussion'"
date: 2019-10-14T18:00:00-04:00
draft: false
menu: main

> authors: ["Kelsey Breseman", "Data Together"]

Reviewer: ```suggestion
authors: ["Kelsey Breseman", "Kevin Nguyen", "Data Together"]
```

---

*Data Together is a community of people building a better future for data. We engage in a monthly [Reading Group](https://github.com/datatogether/reading_datatogether) on themes relevant to information and ethics. Participants' backgrounds range decentralized web protocols, data archiving, ethical frameworks, and citizen science.*

*This reading group is something your own collective can do too! We encourage you to draw on [our notes document template for this month's topic](https://hackmd.io/kKt8JbHvSzC6NnK6OgccWQ). The template lists readings, calls out themes, and suggests discussion questions.*

*This blog post is derived from our conversation, but is not a replica of it; we rearrange and paraphrase throughout. See the [recorded call](https://youtu.be/fbJUSvO_Xvc) for the full discussion!*

---

> This month's reading selections begin with traditional notions and practices of stewardship: Pastor Henry Wright's sermon [The Stewardship of Time](https://youtu.be/RWCbK8qRkuo?t=170); selections from [Haa Tuwunáagu Yís](https://www.goodreads.com/book/show/2146282.Haa_Tuwun_agu_Y_s_for_Healing_Our_Spirit), a collection of Tlingit narratives compiled by Nora Marks Dauenhauer; and [Tending the Wild](https://www.ucpress.edu/book.php?isbn=9780520280434) by Kat Anderson, a view into Ohlone land stewardship practices.

Reviewer: The video for the sermon was removed =(
Suggest we link to transcript instead
(Really wish I had downloaded the audio, darn)

```suggestion
This month's reading selections begin with traditional notions and practices of stewardship: Pastor Henry Wright's sermon [The Stewardship of Time](https://docs.google.com/document/d/1l8tfRlAAL6y4JDRpeK4s--NTRcdBitMHpjsVfAzbPtU/edit); selections from [Haa Tuwunáagu Yís](https://www.goodreads.com/book/show/2146282.Haa_Tuwun_agu_Y_s_for_Healing_Our_Spirit), a collection of Tlingit narratives compiled by Nora Marks Dauenhauer; and [Tending the Wild](https://www.ucpress.edu/book.php?isbn=9780520280434) by Kat Anderson, a view into Ohlone land stewardship practices.
```

Author: Oh no! Thanks for checking the link though.

Finally, we finish with selections from Nadia Eghbal's report [Roads and Bridges: The Unseen Labor Behind Our Digital Infrastructure](https://www.fordfoundation.org/media/2976/roads-and-bridges-the-unseen-labor-behind-our-digital-infrastructure.pdf) on the problems facing open-source software and how to sustain digital infrastructure.

These readings explore the definition of stewardship; proper use as a form of caretaking; longevity and the need for institutions; the necessity of curation; and care for the maintainers and stewards themselves.

Please see the bottom of the post for a full list of reading selections.

---
## Defining stewardship

**KEVIN**: Before we talk about *data* stewardship, **how do each of you define "stewardship"? Where did you get those ideas from, and how did they evolve?**

For example, my religious background was my first exposure to the idea of stewardship.

> *From the [Wright sermon](https://youtu.be/RWCbK8qRkuo?t=170):*

Reviewer: ```suggestion
*From the [Wright sermon](https://docs.google.com/document/d/1l8tfRlAAL6y4JDRpeK4s--NTRcdBitMHpjsVfAzbPtU/edit):*
```

My other context is **land stewardship**. My dad spent his whole career in timber and timber sales for Alaska Native corporations. These native corporations are set up to steward the resources of the tribe in perpetuity. Tribal members are shareholders, but it's not just an economic relationship. It's also supposed to provide for recreation, spiritual practice, forage– forever.

**ERIC**: Kelsey raised the issue of ownership when it comes to stewardship. According to the pastor in our video, a steward is *not* an owner. A steward is taking care of somebody else's stuff.

The pastor's comments define **different dimensions of stewardship: ownership, accountability, responsibility, authority**. Stewardship involves taking care of something so that others may benefit.

**ROB**: The sermon captured my concept of stewardship: taking care of and being responsible for somebody else's stuff.

## Motives for stewardship

> **KEVIN**: I liked that people touched on ownership. I was chatting with my neighbor, who is an Episcopelian pastor. They had a great term, that it's the "antidote to dominion". **Instead of lording over or controlling things, a steward is involved with the proper care and *use* of things.**

Reviewer: ```suggestion
**KEVIN**: I liked that people touched on ownership. I was chatting with my neighbor, who is an Episcopalian pastor. They had a great term, that it's the "antidote to dominion". **Instead of lording over or controlling things, a steward is involved with the proper care and *use* of things.**
```

## Closing

**KELSEY**: We started by talking about stewardship in several contexts, often with physical aspects. I think that helped ground us as we moved through a digital-focused conversation around what we're keeping, why we're keeping it, and who we're keeping it for.

I'd like to close with quote from a favorite short story, which comes back to not just physical artifacts, but experiential, and thus the inevitable disappearance of data over time.

This is from "[The Witness](https://thefloatinglibrary.com/2008/09/14/the-witness/)", by Jorge Luis Borges:

> But something, or an infinite number of things, dies in every death, unless the universe is possessed of a memory, as the theosophists have supposed.
> In the course of time, there was a day that closed the last eyes to see Christ. The bottle of Junin, and the love of Helen each died with the death of some one man. What will die with me when I die, what pitiful or perishable form will the world lose? The voice of Macedonio Fernandez? The image of a roan horse on the vacant lot at Serrano and Charcas? A bar of sulfur in the drawer of a mahogany desk?

> **KEVIN**: That's a great way to leave us thinking. I hope people think about how to continue this conversation: defining how to be good stewards, and deciding what should be preserved, what we want to preserve, or letting people choose. Thanks for an awesome conversation.

Reviewer: ```suggestion
**KEVIN**: That's a great way to not end the conversation, but leave us thinking. I hope people think about how to continue this conversation: defining how to be good stewards, and deciding what should be preserved, what we want to preserve, or letting people choose. Thanks for an awesome conversation.
```

---

**Readings**

> 1. Pastor Henry Wright, [The Stewardship of Time](https://youtu.be/RWCbK8qRkuo?t=170), 2019: 2:50-6:17. [Transcript](https://docs.google.com/document/d/1l8tfRlAAL6y4JDRpeK4s--NTRcdBitMHpjsVfAzbPtU/edit)

Reviewer: ```suggestion
1. Pastor Henry Wright, The Stewardship of Time, 2019: 2:50-6:17. [Transcript](https://docs.google.com/document/d/1l8tfRlAAL6y4JDRpeK4s--NTRcdBitMHpjsVfAzbPtU/edit)
```
